source/sass/buyers-guide/views/homepage.scss

```diff
@@ -24,20 +24,7 @@
         font-size: 20px;
       }
 
-      .creep-face-emoji {
-        display: inline-block;
-        width: 1em;
-        height: 1em;
-        position: relative;
-        top: 0.25em;
-        margin: 0 0.2em;
-        font-variant: normal;
-        color: transparent;
-        background-image: url(/_images/buyers-guide/faces/most-emoji.png);
-        background-size: 100% 100%;
-        background-position: center center;
-        background-repeat: no-repeat;
-      }
+      @import "../creep-face-emoji";
```

Comment:
<img width="422" alt="screen shot 2018-12-19 at 10 45 48 am" src="https://user-images.githubusercontent.com/2896608/50241418-50ec4400-037c-11e9-97d3-af2784166ea2.png">

maybe as a followup? can we make sure the selector is not so long and specific?